content/pages/info.rst

```diff
@@ -6,7 +6,7 @@ WiFi
 ----
 
 We are using the OSU public wifi system. Look for **OSU_ACCESS** (choose local
-DB), account name is **BBC14**, password is **sleepyMandrill** (this account
+DB), account name is **BBC11**, password is **bossyPeccary** (this account
```

Comment:
So I think they did away with conference wifi addresses. We should update the instructions to having folks connect to the Visitor SSID and link this page: http://oregonstate.edu/helpdocs/network/wireless/getting-connected